CoEpi/storage/CEN.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import CryptoSwift
+import RealmSwift
+
+struct CEN : Codable {
+    var CEN: String
+    var timestamp: Int = Int(Date().timeIntervalSince1970)
+    var latitude: Double? = nil
+    var longitude: Double? = nil
+    
+    static func generateCENData(CENKey : String, currentTs : Int)  -> Data {
```

Comment:
Should these be static? Feels like there should be a `CENRepo` object that's an injected singleton at the Dip layer, and it can govern these CEN CRUD operations

Reply:
Received the same feedback from @i-schuetz ... not familiar with the Repo model, but he discussed and am working on modifying now.